spec/controllers/verify/confirmations_controller_spec.rb

```diff
@@ -34,7 +34,7 @@
       let(:saml_authn_request) { sp1_authnrequest }
 
       before do
-        subject.session[:saml_request_url] = saml_authn_request
+        subject.session[:user_return_to] = saml_authn_request
```

Comment:
After digging some more, it appears that the `#continue` method is not used, so any  specs that test `post :continue` should be removed, as well as the method.

The `index` action of `Verify::ConfirmationsController` is the recovery code view, which doesn't make any calls to `Verify::ConfirmationsController#continue`.